pallet/xrpl-bridge/src/lib.rs

```diff
@@ -253,15 +260,30 @@ pub mod pallet {
 	#[pallet::getter(fn process_xrp_transaction_details)]
 	/// Stores submitted transactions from XRPL waiting to be processed
 	/// Transactions will be cleared according to the submission window after processing
-	pub type ProcessXRPTransactionDetails<T: Config> =
-		StorageMap<_, Identity, XrplTxHash, (LedgerIndex, XrpTransaction, T::AccountId)>;
+	pub type ProcessXRPTransactionDetails<T: Config> = StorageMap<
+		_,
+		Identity,
+		XrplTxHash,
+		(LedgerIndex, XrpTransaction<T::XRPLTransactionLimitPerLedger>, T::AccountId),
+	>;
 
 	#[pallet::storage]
 	#[pallet::getter(fn settled_xrp_transaction_details)]
 	/// Settled xrp transactions stored against XRPL ledger index
 	pub type SettledXRPTransactionDetails<T: Config> =
 		StorageMap<_, Twox64Concat, u32, BoundedVec<XrplTxHash, T::XRPLTransactionLimitPerLedger>>;
 
+	/// Map GA asset Id to ERC20 address
```

Comment:
Nit. GA is a CENNZnet thing
```suggestion
	/// Map TRN asset Id to ERC20 address
```